client/mapView/MapView.cpp

```diff
@@ -210,7 +210,12 @@ void MapView::onMapZoomLevelChanged(int stepsChange, bool useDeadZone)
 void MapView::onViewMapActivated()
 {
 	controller->activateAdventureContext();
-	controller->setTileSize(Point(32, 32));
+
+	int zoom = persistentStorage["tileZoom"].Integer();
```

Comment:
Why not in settings, as usual?
```suggestion
	int zoom = persistentStorage["tileZoom"].Integer();
```

Reply:
tutorial makes sense. But kingdom overview & this one - not really. Should be in settings

Reply:
Hmm...

- Is this related to game progress? Like completed campaigns? Or completed tutorial? To persistent.

- Is this something that can be changed by player explicitly at least in theory? To settings. Like last kingdom interface - there is nothing wrong with having player select this in UI. Sure, basically nobody would need it, and it would make things harder to find, but there is nothing wrong with making this option configurable